src/control/cmd/daos/pool.go

```diff
@@ -8,6 +8,7 @@ package main
 
```

Comment:
Noting it here because it is slightly related. This code covers the "daos pool list-containers" command
But a while ago we had prepared for an eventual (comparable/identical) "dmg pool list-containers" command too. This would use dRPC infrastructure implemented in this commit https://github.com/daos-stack/daos/commit/8da59333dd0330e8dd6855ae085fab28c8975f02 that otherwise is not exercised much outside of unit testing. 

In this patch I have not updated that portion of the code. Perhaps up for discussion now - should it be? I'm proposing we address this patch in its current scope and separaetly consider when to expand the dRPC interface in preparation for dmg tool support.

Reply:
I agree that it's probably best to land this first, then look at adding the dmg side.